File: documentation-examples-java/src/java/dev/typr/foundations/docs/oracle/XmlJsonTypes.java
package dev.typr.foundations.docs.oracle;

import dev.typr.foundations.OracleType;
import dev.typr.foundations.OracleTypes;
import dev.typr.foundations.data.Json;

@SuppressWarnings("unused")
public class XmlJsonTypes {
    //start
    OracleType<String> xmlType = OracleTypes.xmlType;
    OracleType<Json> jsonType = OracleTypes.json;

    Json data = new Json("{\"name\": \"Oracle\"}");
    //stop
}

File: documentation-examples-scala/src/scala/dev/typr/foundations/docs/oracle/XmlJsonTypes.scala
package dev.typr.foundations.docs.oracle
import dev.typr.scalafoundations.*
import dev.typr.scalafoundations.data.*



@SuppressWarnings(Array("unused"))
object XmlJsonTypes:
  //start
  val xmlType: OracleType[String] = OracleTypes.xmlType
  val jsonType: OracleType[Json] = OracleTypes.json

  val data: Json = new Json("{\"name\": \"Oracle\"}")
  //stop
